### PR TITLE
Local highscore for web

### PR DESCRIPTION
**Pull request by tguelenman**
this seems to work... cant test a lot on web currently with no controls

btw score isnt showing ingame, but this was already the case on master (tested!)

### CODE REVIEW

File: lib/game.dart [hunk 1]
```diff
@@ -23,6 +23,7 @@ class CoronaJump extends BaseGame with HasWidgetsOverlay {
   LevelWrapper level;
   double maxHeight = 0;
   int score = 0;
+  int localHighScore = 0;
```
Comment (abegehr) [inline lib/game.dart]: To keep things clean (and make the code platform agnostic in game and both overlays), can the current highscore be saved in the score helper?

Comment (abegehr) [reply lib/game.dart:26]: @tguelenman, you must have missed this.

Comment (tguelenman) [reply lib/game.dart:26]: @abegehr sorry, missed it indeed. Same thing here though, can we just leave it like this? If I'm totally wrong and this project has a future, then I'll fix it with pleasure

**Comment by abegehr** [comment]
@tguelenman this was approved.

**Comment by abegehr** [comment]
@tguelenman, master needs to be merged into coronajump after merging this into master, for these changes to reflect on CoronaJump as well

**Comment by abegehr** [comment]
@tguelenman this is still open

**Comment by tguelenman** [comment]
@abegehr why arent you merging this after approving?

**Comment by abegehr** [comment]
@tguelenman I do not merge, I just approve.

**Comment by tguelenman** [comment]
@abegehr but why?

**Comment by abegehr** [comment]
@tguelenman merging is the responsibility of the person opening the PR. If a review is requested from me, I review it. If a merge is requested from me, I merge it (if satisfactory).

**Comment by tguelenman** [comment]
@abegehr i dont mind it, but it's not how we've handled things most of the time. why the change of mind?

**Comment by abegehr** [comment]
@tguelenman, you and Adrian are capable of taking more ownership of the code. This includes reviewing your own code before requesting a review and merging your own code.

**Comment by tguelenman** [comment]
@abegehr ok, if that's where you see a change, cool 👍

**Pull request merged**